stream.go

```diff
@@ -0,0 +1,150 @@
+package tarantool
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+type TxnIsolationLevel uint64
+
+const (
+	// By default, the isolation level of Tarantool is serializable.
+	defaultIsolationLevel = iota
+	// The ReadCommittedLevel isolation level makes visible all transactions
+	// that started commit (stream.Do(NewCommitRequest()) was called).
+	ReadCommittedLevel
+	// The ReadConfirmedLevel isolation level makes visible all transactions
```

Comment:
Shouldn't we have tests on different isolation levels? Or am I understanding concept wrong and we couldn't manually change the level?

Reply:
https://github.com/tarantool/go-tarantool/pull/185#discussion_r929729381

Reply:
Should we add a test for each isolation level or it isn't really worth it?

Reply:
I guess it would be fair to run `Begin` with every isolation level, here is the test:

https://github.com/tarantool/go-tarantool/blob/3c3e3b68515ad7e953e37024698dd7699618eb88/tarantool_test.go#L2551-L2643

Reply:
Ok, let it be like this